### ISSUE TITLE
My classes. 

### ISSUE THREAD

**Issue by luizhmatoso** [al-caughey/Google-Meet-Attendance#20]
Hey, 
I didn't understand what"my classes" is for. Can't I have a file with all my classes and just choose what file I want to use in that class?

**Comment by Eagle-Man** [comment]
"My Classes" is a header for where your class lists reside.  This is opposed to the default list "Class List,"  and the "Other Options" of Add and Reset.

If I understand you correctly, you have your class lists stored in files, and then you load them via the folder icon for each class?  If so, that is certainly one way to use the extension.

Another way is, if you Add a new class (under Other Options), you can name it and place the class list in that list.  Do this for all your classes.  Then you only need to select the appropriate list from the drop down menu instead of loading it in each time.

This also has an added benefit of including the class name in the Excel file, both in the first cell and in the file name, so you know which class the attendance is for.  Check marks will also be persistent until you clear them, so if you forget to export your attendance for one class, you can still go back to it in the drop down menu and see the last results.

The folder icon is mostly for importing your class lists the first time you set them up, and possibly again if your browser happens to lose the data, but it should be used infrequently.

Notes:
1) Class names cannot be edited once set, and they are sorted alphabetically (numbers first), so if you want them in another order, you will need to prepend a letter or number in the correct sequence.
2) If your browser or device is managed by your organization, it's possible they have disabled LocalStorage in the browser.  If sync is also not enabled, then your lists will not be saved between browser sessions, and therefore, loading with the folder icon is the best way to go.  To use My Classes, either LocalStorage or Chrome Sync must be enabled (if I understand Al correctly).

**Comment by luizhmatoso** [comment]
Every time I join a meeting, my previously class appear to me, but, I don't know why, I can't see all my classes. It seems I don't have any classes lists. Do you know why?

**Comment by Eagle-Man** [comment]
Have you created separate classes using the Add feature?  If not, then you will only ever see the last list you used.  Have you checked out [Al's tutorial video](https://www.youtube.com/watch?v=dk6brd7y1QY)?  Adding (creating) classes is around the 0:50 mark.  It looks slightly different now, but the steps are still the same.

**Comment by luizhmatoso** [comment]
Look, I put 2 lists to make a test. 
The lists' names don't appear to me. 
![image](https://user-images.githubusercontent.com/65918882/82839708-a64c4f00-9ea6-11ea-868e-a08fe8426517.png)

That's why I'm not understanding

**Comment by luizhmatoso** [comment]
And if I put "add" it doesn't allow me to take a file, but just write

![image](https://user-images.githubusercontent.com/65918882/82840121-df38f380-9ea7-11ea-932c-470abd7db511.png)

**Comment by luizhmatoso** [comment]
And also, I cant see what I have already put 
![image](https://user-images.githubusercontent.com/65918882/82840359-9c2b5000-9ea8-11ea-8469-5e83c3ed8081.png)

**Comment by Eagle-Man** [comment]
Very strange.  Possibly an issue with localization?  @al-caughey?

**Comment by luizhmatoso** [comment]
Do you know how can I enable the localstorage and the chrome sync?

**Comment by Eagle-Man** [comment]
[Enabling LocalStorage](https://voicethread.com/howto/enabling-cookies/)
[Enabling Chrome Sync](https://support.google.com/chrome/answer/185277?co=GENIE.Platform%3DDesktop&hl=en-US)

**Comment by Eagle-Man** [comment]
@luizhmatoso Any update?  Still having this issue?

**Comment by luizhmatoso** [comment]
Yes, man. I still have the same problem, It doesn't save my classes, I have to put the name of my guests every time I start a meeting. ;(

**Comment by al-caughey** [comment]
@luizhmatoso - thanks for your patience... I think you issue might be fixed now.  You were not using English... right?  Someone else just pointed out a major gaff in my logic (I was only looking for `My Classes` in English and forgot to change that when I added the translation.

I will pushed the correction shortly.

**Comment by al-caughey** [comment]
You have (or should get) version 0.6.1 shortly... let me know ASAP if the problem persists

**Comment by luizhmatoso** [comment]
Yes, I am from Brazil. I was thinking the problem was something about my G Suite settings. Thank you so much for the attention. I am going to share this extension with my whole school LOL. Thanks again.

**Comment by al-caughey** [comment]
Sorry, I just noticed this now.  Sorry for missing it.

Did you also have the Meet Attendance extension installed?  It used a CSS library that alters the expected display characteristics of HTML elements (including hiding things and preventing clicks?!?)

Disable or remove extension by claycodes.org to fix the problem... I tried to change my CSS to override his library but it was a never ending can of worms.
